python/vyos/ifconfig/interface.py

```diff
@@ -974,6 +984,11 @@ def update(self, config):
         value = '1' if (tmp != None) else '0'
         self.set_proxy_arp_pvlan(value)
 
+        # IPv4 forwarding
+        tmp = vyos_dict_search('ip.disable_forwarding', config)
+        value = '0' if (tmp != None) else '1'
```

Comment:
I have a strong feeling the logic is inverted here. Merging this will kill all forwarding as 1 will be written to sysctl when disable forwarding is not set - which is de default. This will suspend routing on every interface